manifests/m/Microsoft/PowerShell/7.4.0/Microsoft.PowerShell.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.PowerShell
+PackageVersion: 7.4.0

```

```suggestion
PackageVersion: 7.4.0.0
```

PackageVersion should match the version written to Control Panel / Registry. This is to help WinGet do correct package correlation / upgrade matching